src/main/xsd/portal-signature-job.xsd

```diff
@@ -42,8 +42,8 @@
     <xs:element name="portal-signature-job-status-change-response">
         <xs:complexType>
             <xs:sequence>
+                <xs:element name="signature-job-id" minOccurs="1" maxOccurs="1" type="common:signature-job-id"/>
```

Comment:
Er det noe vits å kalle den `signature-job-…`? Kanskje greit for å være eksplisitt, men det er vel ganske entydig.